hook!-full-game/addons/level_design_dock/LevelDesignDock.gd

```diff
@@ -0,0 +1,27 @@
+tool
+extends Control
+
+signal packed_scene_button_created(button)
+onready var grid_container: GridContainer = $VBoxContainer/GridContainer
+
+const PACKED_SCENE_BUTTON: = preload("res://addons/level_design_dock/PackedSceneButton.tscn")
```

Comment:
When loading tscn and gd files, we should think about a convention. They can both be considered classes. I'd write `PackedSceneButtonScene` and `PackedSceneButton`. I'll write it in the discord chat so we can discuss it.

Reply:
How about it? Do we have any convention already? I think PascalCase + `Scene` suffix sounds good.

Reply:
Let's go with that then